src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -1591,10 +1605,22 @@ GenTree* Compiler::impHWIntrinsic(NamedIntrinsic        intrinsic,
     }
 
 #if defined(TARGET_ARM64)
-    if (HWIntrinsicInfo::IsMaskedOperation(intrinsic))
+    if (HWIntrinsicInfo::IsExplicitMaskedOperation(intrinsic))
     {
         assert(numArgs > 0);
         GenTree* op1 = retNode->AsHWIntrinsic()->Op(1);
+        if (intrinsic == NI_Sve_ConditionalSelect)
```

Comment:
Not important for this PR, but this is potentially something that should be handled in `gtNewSimdCndSelNode` instead and then more generally as part of `morph` to capture values that don't materialize as constants until later.

Reply:
will do it in follow-up PR.